packages/css/src/details.css

```
@@ -1,5 +1,6 @@
 .ds-details {
-  --dsc-details-border-block-wdith: var(--ds-border-width-default);
+  --dsc-details-border-block-wdith: var(--ds-border-width-default); /*@todo: decide how to deprecate this typo after https://github.com/digdir/designsystemet/issues/3297 */
```

```suggestion
  --dsc-details-border-block-wdith: var(--ds-border-width-default); /* TODO: remove next major. Decide how to deprecate this typo after https://github.com/digdir/designsystemet/issues/3297 */
```

Suggest using common syntax for `TODO`s. 

Easier to track with [extensions](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree).